HPXMLtoOpenStudio/resources/hvac_sizing.rb

```diff
@@ -904,12 +904,13 @@ def self.process_infiltration_ventilation(model, zone_loads, weather, infilvolum
     icfm_Cooling = ach_Cooling / UnitConversions.convert(1.0, 'hr', 'min') * infilvolume
     icfm_Heating = ach_Heating / UnitConversions.convert(1.0, 'hr', 'min') * infilvolume
 
-    q_unb, q_bal_Sens, q_bal_Lat = get_ventilation_rates(model)
+    q_unb_cfm, q_preheat, q_precool, q_recirc, q_bal_Sens, q_bal_Lat = get_ventilation_rates(model)
 
-    cfm_Heating = q_bal_Sens + (icfm_Heating**2.0 + q_unb**2.0)**0.5
+    cfm_Heating = q_bal_Sens + (icfm_Heating**2.0 + q_unb_cfm**2.0)**0.5 - q_preheat - q_recirc
 
-    cfm_Cool_Load_Sens = q_bal_Sens + (icfm_Cooling**2.0 + q_unb**2.0)**0.5
-    cfm_Cool_Load_Lat = q_bal_Lat + (icfm_Cooling**2.0 + q_unb**2.0)**0.5
+    # FIXME: Assume preconditioning doesn't introduce/handle latent loads?
```

Comment:
I would do whatever is easiest since we don't really know if the preconditioning equipment can address the latent load or not.